docker-compose-k6.account.yaml

```diff
@@ -0,0 +1,71 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/compose-spec/compose-spec/master/schema/compose-spec.json
+
+services:
+  account-service-api:
+    pull_policy: never
+    build:
+      context: .
+      dockerfile: ./Docker/Dockerfile.account
+    command: ['api']
+    ports:
+      - 3000:3000
+    volumes: !reset []
+    depends_on: !override
+      redis:
+        condition: service_started
+      ipfs:
+        condition: service_healthy
+      account-service-webhook:
+        condition: service_healthy
+    environment:
+      PROVIDER_BASE_URL: 'http://account-service-webhook:3001/webhooks/account-service'
+      WEBHOOK_BASE_URL: 'http://account-service-webhook:3001/webhooks'
+
+  account-service-worker:
+    pull_policy: never
+    build:
+      context: .
+      dockerfile: ./Docker/Dockerfile.account
+    command: ['worker']
+    volumes: !reset []
+    depends_on: !override
+      - redis
+      - ipfs
+    environment:
+      PROVIDER_BASE_URL: 'http://account-service-webhook:3001/webhooks/account-service'
+      WEBHOOK_BASE_URL: 'http://account-service-webhook:3001/webhooks'
+
+  account-service-webhook:
+    image: rust:1.80.0
+    volumes:
+      - ./services/account/rust-webhook-server:/app
+    ports:
+      - '3001:3001'
+    command: sh -c "cd /app && cargo run"
+    healthcheck:
+      test: ['CMD', 'curl', '-f', 'http://127.0.0.1:3001/webhooks/health']
+      interval: 30s
+      timeout: 10s
+      retries: 3
+      start_period: 40s
+    networks:
+      - amplica-gateway
+
+  gateway-base:
+    profiles:
+      - skip
```

Comment:
This is so these services don't start by default. Easiest way to do this I found